bundles/org.openhab.binding.bsblan/src/main/java/org/openhab/binding/bsblan/internal/api/models/BsbLanApiParameterSetResponse.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.bsblan.internal.api.models;
+
+import org.openhab.binding.bsblan.internal.api.BsbLanApiContent;
+
+import java.util.HashMap;
+
+/**
+ * The {@link BsbLanApiParameterSetResponse} reflects the response received
+ * when setting a parameter.
+ *
+ * @author Peter Schraffl - Initial contribution
+ */
+@SuppressWarnings("serial")
+public class BsbLanApiParameterSetResponse extends HashMap<Integer, BsbLanApiParameterSetResult> implements BsbLanApiContent {
```

Comment:
Why do you need this class?

Reply:
It is used as a data model for the REST API response received when changing parameter values.